### DISCUSSION TITLE
Semver, with prefix

### DISCUSSION THREAD

**Discussion by cypher7682** [renovatebot/renovate#26016] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovatebot/github-action@v36.0.3

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

We have a github repo that releases with a prefix on the regular semver, as it contains multiple submodules; like so:

- `module1-v1.0.0`
- `module1-v1.0.1`
- `module2-v2.0.0`
- `module2-v2.0.1`

Can we get renovate to honor the prefix on those releases?

At the moment it _does_ seem to notice that it's called `module1-v8.0.1`, but wants to upgrade to a version that _doesn't_ have the prefix. Keep in mind that this module is part of a load of other modules that release using standard semver releases as `vX.Y.Z`.

**Comment by rarkins** [comment, marked as answer]
Yes, configure versionCompatibility for that package
